src/roles/common/tasks/check-predeploy-prereq.yml

```diff
@@ -45,7 +67,7 @@
         msg: "Missing VRS DKMS files under the unzipped file directory, please unzip it."
```

Comment:
`that: not is_dkms_package_missing`

You can also remove the when: since dkms_install is covered by the set_fact.

Reply:
done